mmdet/core/bbox/registry.py

```diff
@@ -0,0 +1,4 @@
+from mmdet.utils import Registry
+
+ASSIGNERS = Registry('assigner')
+SAMPLERS = Registry('sampler')
```

Comment:
Rename the registies to `BBOX_ASSIGNERS` and `BBOX_SAMPLERS` to avoid ambiguity. There is also a registry for dataset sampler.